### ISSUE TITLE
Tags not saving in order, and more.

### ISSUE THREAD

**Issue by MoeMonsuta** [HaW-Tagger/HWtagger#3]
Not sure if this project going still but I noticed bugs with the installer and the program itself. Managed to get through dependency hell and get it going and it works ok, but toml doesn't work at all (I might be doing that wrong though) and also the color coded tags like in your pictures are missing. Tag txt export also puts all the tags out of order. 

EDIT: Had to completely reinstall using my own made venv because the installer one had the torch version I needed installed REM'ed out. Managed to fix all the issues I was having (mainly from cuda/cudnn/onnxruntime not installing right even after I reinstalled), but **the tagger txt export still is not doing it in the correct order that shows in the Hecate Tagger window.**

Example is easier to understand.

The sample image
![2024-10-05_213850](https://github.com/user-attachments/assets/5be138ec-2f45-4a3e-915d-11c9f1d6d266)

Tag order is perfect here.
![2024-10-05_214111](https://github.com/user-attachments/assets/998ee79f-026b-41f0-8d50-dcc681295ffe)

Tag order is completely shuffled up!
![2024-10-05_213858](https://github.com/user-attachments/assets/e5dfe4f3-5bd3-48ee-a312-4b3dc933df4a)

If there is a setting to I need to check please let me know if I'm doing this wrong. 

Also note the https://medium.com/geekculture/install-cuda-and-cudnn-on-windows-linux-52d1501a8805#3e72 link is dead.

**Comment by PhoenixAscencio** [comment]
Hello, the project is not dead, it's just that we started doing a huge update and basically until everything is done properly we won't be able to merge into the base code (We are probably gonna make a dev branch though because the current state is quite stable just that the new feature are not complete or properly documented/tested yet).

In the new version we made an installer with conda that takes care of CUDNN itself and should be easier for everyone but it will take more disk space if you have multiple software that uses the CUDNN libraries. 
The tag order is randomized on purpose (for training it's better but we can add a checkbox to deactivate it if needed), We can create a button to copy paste the tags with the current order on top.
